src/client/utils/prismjs/PrismCodeParser.js

```diff
@@ -0,0 +1,194 @@
+/* flow */
+
+import Prism from 'prismjs'
+
+const REMOVE_MARK = '-'
+const ADDED_MARK = '+'
+
+/* eslint-disable */
+
+export const getLineNumbers = (line) => {
```

Comment:
That's fairly simple parsing of raw diff after the tokenizing piece of code by prismjs lib. Then I need to identify where the new and old lines two different ways for unified and side-by-side.  Could be difficult on the first glance but we could add annotations and some unit test for this.

But what kind of issues it has?